lang/gtext_eng.pot

```diff
@@ -1361,8 +1361,8 @@ msgstr ""
 #: guitext:156
 msgctxt "In-game message"
 msgid ""
-"Your evil has corrupted a Samurai Warrior, he has forsaken honour to join "
-"your cause. Should your minions suffer heavy casualties, this Warrior will "
+"Your evil has corrupted a Samurai warrior; he has forsaken honour to join "
```

Comment:
Is a period not a more elegant solution here?

Reply:
It looks to me like the two clauses are closely related enough to warrant a semicolon. Why do you think they should be discrete sentences?

Reply:
semicolon don't looks very good in game

Reply:
> 

![image](https://github.com/dkfans/keeperfx/assets/58278560/85cd27d5-a5e5-489a-b82e-4e579f6150b1)

Looks okay to me 🤔.

Reply:
A full colon would also work here, though that depends on whether we want to emphasise the second clause.